context/app/static/js/components/Detail/ProvTable.jsx

```diff
@@ -60,7 +60,7 @@ function ProvTable(props) {
             )
         }
             {
-          typesToSplit[i] === entity_type && type !== 'Donor'
+          typesToSplit[i] === entity_type && typesToSplit[i] !== 'Donor'
```

Comment:
perhaps
```suggestion
          typesToSplit[i] === entity_type && entity_type !== 'Donor'
```
but whichever you think is more clear.

Reply:
`entity_type` refers to the type for the detail page that is being viewed. The suggested change would cause a donor detail page to not render derived samples or datasets.

Reply:
I could rename the variable to be more clear?

Reply:
huh: Then I'm confused: From the first part, we know that that `typesToSplit[i] === entity_type`.... so in the second we should be able to substitute `entity_type` for `typesToSplit[i]` without changing the semantics? I may be missing something.

Reply:
You're right, my mistake.